public/js/studentApplicationForm.js

```diff
@@ -2,7 +2,12 @@
 const prevButtons = document.querySelectorAll('.prevButton')
 const nextButtons = document.querySelectorAll('.nextButton')
 const formWrappers = document.querySelectorAll('.studentApplicationFormPages')
+const progressBar = document.querySelector('#progressBar')
 const pageCounter = document.querySelector('#pageCounter')
+const textAreaToCount = document.querySelector('#whyDev')
```

Comment:
Might be nice if textAreaToCount and textAreaCount indicated which text area they were counting in the variable name :)